app/views/external_users/claims/buttons/_continue.html.haml

```diff
@@ -1,6 +1,7 @@
 .form-section
   .form-group.form-buttons.button-holder
     - commit = claim.next_step? ? 'commit_continue' : 'commit_submit_claim'
-    = govuk_button(t('buttons.continue'), id: 'save_continue', name: commit)
+    .govuk-button-group
+      = govuk_button(t('buttons.continue'), id: 'save_continue', name: commit)
 
-    = govuk_button_secondary(t('buttons.save_a_draft'), name: 'commit_save_draft')
+      = govuk_button_secondary(t('buttons.save_a_draft'), name: 'commit_save_draft')
```

Comment:
This is a very minor thing. Now that the buttons are in a group then the blank line above can be removed. This will make it clearer that these are grouped. I think there may be a couple of other examples of this.